content/gsoc_2023/gsoc_home.md

```diff
@@ -58,7 +59,8 @@ Since 2005, Google has run an awesome open source software development program,
   - That, in turn, determines how many student developers work with the TARDIS collaboration.
 
 Please also read the following pages before applying. Project ideas can be found on the [ideas page](../ideas).
+
 1. [Application Requirements](../requirements)
 2. [Application Guidelines](../guidelines)
 3. [Application Checklist](../checklist)
-5. [Participation Expectations](../expectations)
+4. [Participation Expectations](../expectations)
```

Comment:
These aren't relevant to the pull request, right?

Reply:
No I don't think so. I don't remember making thesw changes. Only aliases. And removing to .md files which were not necessary